wcc-contentful/lib/wcc/contentful/store/query.rb

```diff
@@ -108,7 +108,7 @@ def resolve_includes(row, depth)
       includes = row.try(:includes) || row.try(:[], 1)
       return entry unless entry && depth && depth > 0
 
-      WCC::Contentful::LinkVisitor.new(entry, :Link, :Asset,
+      WCC::Contentful::LinkVisitor.new(entry, :Link,
```

Comment:
Any implications for this?

Reply:
Some implications explained in this commit message https://github.com/watermarkchurch/wcc-contentful/pull/249/commits/2fa1f370707b84f17df52e4df1e95f0dbffbc202

Basically, I made a breaking change to the LinkVisitor.  But I don't consider the LinkVisitor to be part of the Public API of the Gem (it's not mentioned in the readme anywhere as something you can use, more of an internal utility)